inst/rmarkdown/templates/lipics/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,302 @@
+  :::
+  
+  \begin{remark*}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  \end{remark*}
+  
+  ::: {.claim data-latex="\label{testenv-claim}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  
+  \begin{claim*}\label{testenv-claim2}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  \end{claim*}
+  
+  ::: {.proof data-latex=""}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  ::: {.claimproof data-latex=""}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+output:
+  rticles::lipics_article
+  #bookdown::pdf_book:
+  #  base_format: rticles::lipics_article # for using bookdown features like \@ref()
+  #  extra_dependencies: plainurl.bst
+---
+
+# Typesetting instructions -- Summary
+
+LIPIcs is a series of open access high-quality conference proceedings across all fields in informatics established in cooperation with Schloss Dagstuhl.
+In order to do justice to the high scientific quality of the conferences that publish their proceedings in the LIPIcs series, which is ensured by the thorough review process of the respective events, we believe that LIPIcs proceedings must have an attractive and consistent layout matching the standard of the series.
+Moreover, the quality of the metadata, the typesetting and the layout must also meet the requirements of other external parties such as indexing service, DOI registry, funding agencies, among others. The guidelines contained in this document serve as the baseline for the authors, editors, and the publisher to create documents that meet as many different requirements as possible.
+
+Please comply with the following instructions when preparing your article for a LIPIcs proceedings volume.
+
+## Minimum requirements {.unnumbered}
+
+- Use pdflatex and an up-to-date \LaTeX{} system.
+- Use further \LaTeX{} packages and custom made macros carefully and only if required.
+- Use the provided sectioning macros: `\section` (`#` in Markdown), `\subsection` (`##` in Markdown), `\subsubsection` (`###` in Markdown), 
+  `\paragraph`, `\paragraph*`, and `\subparagraph*`.
```

Comment:
This can also be written in markdown 

* `\paragraph` => `####`
* `\paragraph*` => `#### title {.unnumbered}`
* `\subparagraph` => `#####`
* `\subparagraph*` => `##### title {.unnumbered}`

out of curiosity is there a reason not to use those ?